components/Branding/FontSection/FontSection.module.css

```diff
@@ -1,18 +0,0 @@
-.fontsList {
-  list-style: none;
-  width: 100%;
-}
-
-.fontsList > li {
-  margin: 1rem;
-}
-
-.primaryFontFamily div > h6,
-.primaryFontFamily div > p {
-  font-family: var(--primaryFontFamily), sans-serif;
```

Comment:
Nice job on the condition.

Do you see here how it's 2 values separated by a comma?

1. var(--primaryFontFamily)
2. sans-serif

That's what's missing.

Reply:
where to set that variable?

Reply:
will it look like this ? -> 
fontFamily:{
   sans:{
      primaryFontFamily:'"DIN Condensed Bold"'
   }
}

Reply:
```
fontFamily: {
  primary: "DIN Condensed Bold"
},
```

Reply:
got it

Reply:
done certain changes